browser/main/modals/PreferencesModal/index.js

```diff
@@ -99,14 +103,15 @@ class Preferences extends React.Component {
 
     const tabs = [
       {target: 'STORAGES', label: 'Storages'},
-      {target: 'HOTKEY', label: 'Hotkey'},
-      {target: 'UI', label: 'UI'},
+      {target: 'HOTKEY', label: 'Hotkey', Hotkey: this.state.Hotkey},
+      {target: 'UI', label: 'UI', UI: this.state.UI},
       {target: 'INFO', label: 'Community / Info'},
       {target: 'CROWDFUNDING', label: 'Crowdfunding'}
     ]
 
     const navButtons = tabs.map((tab) => {
       const isActive = this.state.currentTab === tab.target
+      const isOk = typeof tab[tab.label] !== 'undefined' && tab[tab.label] !== null
```

Comment:
`isOk` is a little bit simple word. Plz specify more detail for variable name like `isOkToSave` or anything